app/speech/SphinxRecognizer.py

```diff
@@ -9,28 +8,29 @@ def __init__(self, threshold, sample_rate=16000):
         self.sample_rate = sample_rate
         self.threshold = threshold
         dict_name = "cmudict-en-us.dict"
-        self.decoder = Decoder(self.create_config(dict_name))
-        #self.decoder.set_search('keyphrase')
+        self.decoder = pocketsphinx.Decoder(self.create_config(dict_name))
 
     def create_config(self, dict_name):
-        modeldir = "/usr/lib/python3.6/site-packages/pocketsphinx/model"
-        config = Decoder.default_config()
-        config.set_string('-hmm', os.path.join(modeldir, 'en-us'))
+        pocketsphinx_dir = os.path.dirname(pocketsphinx.__file__)
+        model_dir = os.path.join(pocketsphinx_dir, "model")
+        print(model_dir)
```

Comment:
Yes, they are not needed. I used them for debugging. Removing them. Thanks for pointing out.